sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryHelpers.java

```diff
@@ -314,39 +297,25 @@ public TableReference apply(String from) {
     }
   }
 
-  @VisibleForTesting
-  static class CreatePerBeamJobUuid
-      implements SerializableFunction<String, String> {
-    private final String stepUuid;
-
-    CreatePerBeamJobUuid(String stepUuid) {
-      this.stepUuid = stepUuid;
-    }
-
-    @Override
-    public String apply(String jobUuid) {
-      return stepUuid + "_" + jobUuid.replaceAll("-", "");
-    }
+  static String createJobIdToken(String jobName, String stepUuid) {
+    return "beam_job_" + createJobUuid(jobName, stepUuid);
   }
 
-  @VisibleForTesting
-  static class CreateJsonTableRefFromUuid
-      implements SerializableFunction<String, TableReference> {
-    private final String executingProject;
+  static String createJobUuid(String jobName, String stepUuid) {
+    return stepUuid + "_" + jobName.replaceAll("-", "");
+  }
```

Comment:
surely we don't need both of these functions? Can we just inline this in `createJobIdToken` so we know we use the good one, and also use `String.format`?